src/config.ts

```diff
@@ -16,9 +16,14 @@ const TX_MINING_TESTNET_URL = 'https://txmining.testnet.hathor.network/';
 const EXPLORER_SERVICE_MAINNET_BASE_URL  = 'https://explorer-service.hathor.network/';
 const EXPLORER_SERVICE_TESTNET_BASE_URL  = 'https://explorer-service.testnet.hathor.network/';
 
+// Explorer service URL
+const SWAP_SERVICE_MAINNET_BASE_URL  = 'https://atomic-swap-service.hathor.network/';
+const SWAP_SERVICE_TESTNET_BASE_URL  = 'https://atomic-swap-service.testnet.hathor.network/';
+
 class Config {
   TX_MINING_URL?: string;
   TX_MINING_API_KEY?: string;
+  SWAP_SERVICE_BASE_URL?: string;
```

Comment:
:speech_balloon: It would be nice if this config property have more information about its purpose and even better if have example of utilization.
```suggestion
  /**
   * This property is used to reach out the Swap service,
   * it can incorporate the values of `SWAP_SERVICE_MAINNET_BASE_URL`,
   * `SWAP_SERVICE_TESTNET_BASE_URL`, or one defined by the user through
   * the @see setSwapServiceBaseUrl function.
   */
  SWAP_SERVICE_BASE_URL?: string;
```

Reply:
When I added this I noted that it would be important to also add the same kind of documentation to all the other `Config` class variables, and that would make the code polluted with docstrings.

So I believe it's better to keep just the variable declaration here and the full description of its usage on the `getSwapServiceBaseUrl` and `setSwapServiceBaseUrl` methods, as it is now.

Do you agree?

Reply:
Mmm... I'm of the option to keep the documentation, even when it messes with code symmetry. However, it is not a big deal. It is ok to not to have it here.